site/en/docs/android/trusted-web-activity/multi-origin/index.md

```diff
@@ -0,0 +1,179 @@
+---
+layout: "layouts/doc-post.njk"
+title: Multi-Origin Trusted Web Activities
+date: 2020-01-17
+updated: 2020-07-31
+description: How to create one application using Trusted Web Activities that supports opening multiple origins in full-screen.
+authors:
+  - andreban
+---
+
+*If you are new to Trusted Web Activities, you may want to read the
```

Comment:
We have `{% Aside %}` and `{% endAside%}`. Would this be better as one of those?

Reply:
Yes, that's much nicer! Added it.